contracts/Router.sol

```diff
@@ -244,8 +314,8 @@ contract Router is ReentrancyGuard {
         return amountsPledged[_addr];
     }
 
-    function getClaimAmount(address _addr) external view returns (uint256) {
-        return amountsClaimable[_addr];
+    function getTreeClaimableAmount(address _addr) external view returns (uint256) {
```

Comment:
should we rename as `getTREEClaimableAmount()` to align with `claimTREE()`?

Reply:
I'll just camel case Tree in all the function names.  We seem to have more using that format.